lighthouse-core/audits/preload-lcp-image.js

```diff
@@ -110,10 +110,64 @@ class PreloadLCPImageAudit extends Audit {
    * @return {{wastedMs: number, results: Array<{url: string, wastedMs: number}>}}
    */
   static computeWasteWithGraph(lcpNode, graph, simulator) {
-    const simulation = simulator.simulate(graph, {flexibleOrdering: true});
-    // For now, we are simply using the duration of the LCP image request as the wastedMS value
-    const lcpTiming = simulation.nodeTimings.get(lcpNode);
-    const wastedMs = lcpTiming && lcpTiming.duration || 0;
+    const simulationBeforeChanges = simulator.simulate(graph, {flexibleOrdering: true});
```

Comment:
how do you feel about moving this down closer to the other simulations, we don't really do anything with it until then